extractor.py

```diff
@@ -8,24 +8,29 @@
 
 
 # constant
-points_to_mm = float(1000 * 1/2834.64567)
+points_to_mm = float(1000 * 1 / 2834.64567)
+points_to_inches = float(0.013888800555552)
 inch_to_mm = float(25.4)
 
 
 def result_line(filename, height, width):
-    return "{};{:.0f};{:.0f}\n".format(filename, width, height)
+    return f"{filename};{width:};{height:}\n"
 
 
-def pdf_mediabox(filename):
+def pdf_mediabox(filename, metric=True):
```

Comment:
what would be if we want to add more metrics?
pass the flag with default value, then use it to choose what you should use